components/needs_team/needs_team.jsx

```diff
@@ -55,40 +55,39 @@ const UNREAD_CHECK_TIME_MILLISECONDS = 10000;
 
 export default class NeedsTeam extends React.Component {
     static propTypes = {
+        actions: PropTypes.shape({
+            viewChannel: PropTypes.func.isRequired,
+            getMyChannelMembers: PropTypes.func.isRequired
+        }).isRequired,
+        center: PropTypes.element,
         children: PropTypes.oneOfType([
             PropTypes.arrayOf(PropTypes.element),
             PropTypes.element
         ]),
         navbar: PropTypes.element,
+        params: PropTypes.object,
         sidebar: PropTypes.element,
         team_sidebar: PropTypes.element,
-        center: PropTypes.element,
-        params: PropTypes.object,
-        user: PropTypes.object,
-        actions: PropTypes.shape({
-            viewChannel: PropTypes.func.isRequired,
-            getMyChannelMembers: PropTypes.func.isRequired
-        }).isRequired
+        theme: PropTypes.object.isRequired,
+        user: PropTypes.object
     }
 
     constructor(params) {
         super(params);
 
         this.onTeamChanged = this.onTeamChanged.bind(this);
```

Comment:
Can you change these to arrow functions instead of using bind or a static function? It saves us a few lines and some headaches if we just use
```
class NeedsTeam {
    shortcutKeyDown = (e) => {
        ...
    }
}
```

Reply:
According [to this jsperf test](https://jsperf.com/static-vs-prototype-methods-test/2) static methods are in the range of 40-50% faster. Is there a specific headache that would prevent us from wanting to take advantage of the performance boost?

Reply:
Do you happen to know if there's any tests comparing ES6 methods against static functions? I'd be up for trying to use static functions in more places if the improvements are that big, but I don't know how babel compiles classes into pre-ES6 javascript

Reply:
Actually I just realized that that previous test was not well constructed for our use case because it was including the instantiation of the class into each test. In our case the class is already instantiated. I have [re-written the test](https://jsperf.com/mk-static-vs-instance) to be closer to our use case and as you can see the difference is much smaller. Static does win every time but only by 1-7%-ish.

Reply:
Forget it, actually. I just got a few test runs where static was slower. Can't recommend this change. Reverting.

Reply:
Cool, well it's good to know that anyway. We had some ESLint rule at some point that complained when methods didn't reference `this`, so I had always wondered if static functions were better

Reply:
Would you mind changing these to arrow functions and getting rid of the bind statements then?